### PR TITLE
docs: add service mesh blog

### PR DESCRIPTION
**Pull request by lijing-21**
add service mesh blog

### CODE REVIEW

**Review by bzp2010 — APPROVED** [review]

File: website/blog/2021-08-20-Service-grid-solution-based-on-Apache-APISIX.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
+---
+title: "基于 Apache APISIX 的服务网格方案"
+avatar: "https://avatars.githubusercontent.com/u/10428333?v=4"
+author: "张超"
+href: "https://github.com/tokers"
+date: 2021-08-20
+keywords: APISIX,服务网格,网关,控制面,架构
```
Comment (bzp2010) [inline website/blog/2021-08-20-Service-grid-solution-based-on-Apache-APISIX.md]: just like this:
```
keywords:
  - APISIX
  - 服务网格
  - 网关
  - 控制面
  - 架构
```

**Review by bzp2010 — CHANGES_REQUESTED** [review]

File: website/blog/2021-08-20-Service-grid-solution-based-on-Apache-APISIX.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+#### 扩展能力强
+
+Apache APISIX 是完全基于插件进行扩展的，二次开发难度比较低，而且还能通过 Lua、Java、Go 和 WASM 等来开发自定义插件。借助灵活的插件机制，可针对内部业务完成功能定制。
+
+### 架构
+
+![Apache APISIX 的架构图](https://static.apiseven.com/202108/1631065632487-3e46ee68-c1fb-4546-a82f-0f81504ce560.png)
+
+这是 Apache APISIX 的架构图，左边是数据面（Data Plane），右边是管理面（Control Plane），配置中心选择了当下最成熟的基础设施：etcd，通过 watch etcd，拿到路由、上游等数据。
```
Comment (yzeng25) [inline website/blog/2021-08-20-Service-grid-solution-based-on-Apache-APISIX.md]: “...配置中心选择了当下最成熟的基础设施：etcd，通过 watch etcd.......”

“...配置中心选择了当下最成熟的基础设施：etcd。 Apache APISIX 通过 watch etcd......”

File: website/blog/2021-08-20-Service-grid-solution-based-on-Apache-APISIX.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+对于服务网格，大家关注的大多是上述比较基础的功能。但是服务网格的功能是十分强大的，而大家只看到了其中一小部分核心功能。
+
+Apache APISIX 已经支持：
+
+- HTTP 代理、gRPC 的代理
+- TCP 、UDP 代理
+- 流量切分（灰度发布、蓝绿部署）
+- 负载均衡
+- 健康检查
+- 认证（mTLS、JWTToken）
```
Comment (yzeng25) [inline website/blog/2021-08-20-Service-grid-solution-based-on-Apache-APISIX.md]: "JWTToken" to "JWT Token"

File: website/blog/2021-08-20-Service-grid-solution-based-on-Apache-APISIX.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+
+#### Apache APISIX 作为数据面带来的问题
+
+- **问题一：如何及时获取配置变更？**
+
+在服务网格场景下，数据面实例可能有很多，如何保证在实例数特别多时配置变更的下发延时较低？
+
+- **问题二：控制面的选型**
+
+一个完整的服务网格方案必须有数据面和控制面，Apache APISIX 可以作为数据面，控制面该怎么选呢？是选现有的服务网格产品，如：Istio / Kuma / OpenServiceMesh ，还是自研？
```
Comment (yzeng25) [inline website/blog/2021-08-20-Service-grid-solution-based-on-Apache-APISIX.md]: "如：Istio / Kuma / OpenServiceMesh"

"如：Istio、Kuma、OpenServiceMesh"

File: website/blog/2021-08-20-Service-grid-solution-based-on-Apache-APISIX.md [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+
+如果南北向流量和东西向流量的技术栈无法统一，需要在东西向上使用 Envoy，在南北向上使用 Apache APISIX，运维团队需要同时维护 2 个组件，这对运维人员来说是一个挑战。
+
+使用基于 Apache APISIX 的服务网格方案，能够统一南北向和东西向的技术栈，只需要管理 1 个组件，大大降低了运维成本。
+
+## 基于 Apache APISIX 的服务网格，规划与发展方向
+
+截止目前，apisix-mesh-agent 开源仅两个月时间，还有许多功能要完善，以下是对 Apache APISIX 服务网格未来的一些规划。
+
+### 第一，对 xDS 协议做更多支持
```
Comment (yzeng25) [inline website/blog/2021-08-20-Service-grid-solution-based-on-Apache-APISIX.md]: delete "第一，"

File: website/blog/2021-08-20-Service-grid-solution-based-on-Apache-APISIX.md [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+目前，apisix-mesh-agent 只支持了 xDS 协议中的 LDS、RDS、CDS 和 EDS，而且只使用 LDS 绑定RDS 需要的路由配置名称，直接获取路由；使用 CDS 和 EDS 直接获取服务集群的定义和服务集群的实例。
+
+未来我们计划：
+
+- 支持 Apache APISIX 更多数据类型的转换
+- 支持请求改写，如：URI 改写
+- 支持请求认证，如 JWT Token、mTLS
+- 支持故障注入
+
+### 第二，提升可观测性
```
Comment (yzeng25) [inline website/blog/2021-08-20-Service-grid-solution-based-on-Apache-APISIX.md]: delete "第二，"

File: website/blog/2021-08-20-Service-grid-solution-based-on-Apache-APISIX.md [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+
+在调研阶段发现，实际使用场景中大家十分重视服务的可观测性，未来支持对接更多可观测方面的组件，如：Apache SkyWalking、其他日志收集的服务器。
+
+### 第三，自研控制面
```
Comment (yzeng25) [inline website/blog/2021-08-20-Service-grid-solution-based-on-Apache-APISIX.md]: delete "第三，"

File: website/blog/2021-08-20-Service-grid-solution-based-on-Apache-APISIX.md [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+
+现在服务网格的控制面依托其他开源组件（如：Istio、Kuma），使得我们可以以较低的成本使用现有的功能，而如果未来遇到问题很难解决，也会考虑去落地实现一个专属 Apache APISIX 的服务网格控制面 ，使 Apache APISIX 的能力更好地发挥出来，服务网格在使用中也更加高效、简便。
```
Comment (yzeng25) [inline website/blog/2021-08-20-Service-grid-solution-based-on-Apache-APISIX.md]: Istio、Kuma ---> Istio 和 Kuma

**Review by yzeng25 — CHANGES_REQUESTED** [review]

**Review by yzeng25 — APPROVED** [review]

File: website/blog/2021-08-20-Service-grid-solution-based-on-Apache-APISIX.md [hunk 9]
```diff
@@ -0,0 +1,209 @@
+---
+title: "基于 Apache APISIX 的服务网格方案"
+avatar: "https://avatars.githubusercontent.com/u/10428333?v=4"
+author: "张超"
+href: "https://github.com/tokers"
+date: 2021-08-20
+keywords:
+  - APISIX
+  - 服务网格
+  - 网关
```
Comment (liuxiran) [inline website/blog/2021-08-20-Service-grid-solution-based-on-Apache-APISIX.md]:
```suggestion
  - API 网关
```

**Review by liuxiran — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Pull request closed without merging**